clusterloader2/pkg/measurement/common/scheduler_latency.go

```diff
@@ -226,7 +226,7 @@ func (s *schedulerLatencyMeasurement) sendRequestToScheduler(c clientset.Interfa
 		responseText = string(body)
 	} else {
 		// If master is not registered fall back to old method of using SSH.
-		if provider == "gke" {
+		if !isSSHSupported {
```

Comment:
Similarly to the etcd measurement, this measurement will have nothing to do if `!(masterRegistered || isSshSupported)`. Can we move this check to the top of the execute method?